test/fixtures/cookbooks/test/recipes/package.rb

```diff
@@ -1,7 +1,17 @@
 # frozen_string_literal: true
 apt_update
 
-haproxy_install 'package'
+case node['platform_family']
+when 'amazon', 'rhel'
```

Comment:
we probably only want this for rhel. If I converge for amazonlinux it can't find the package `haproxy18u`